portfolio/src/main/webapp/teatime.html

```diff
@@ -14,11 +14,52 @@
   </head>
   <body onload="getMessages()">
     <div class="container">
+      <div class="row">
+        <div class="col">
+          <nav class="navbar fixed-top navbar-expand-lg navbar-light">
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+              <div class="navbar-nav justify-content-center">
+                <a class="nav-item nav-link" href="index.html">Trini <span class="sr-only">(current)</span></a>
+                <a class="nav-item nav-link" href="/tea.html">tea</a>
+                <a class="nav-item nav-link active" href="/teatime.html">teatime</a>
+              </div>
+            </div>
+          </nav>
+        </div>
+      </div>
       <div class="row">
         <div class="col">
           <h1>Trini teatime</h1>
         </div>
       </div>
+      <div class="row mb-3">
+        <div class="col">
+          <div class="card">
+            <div class="card-body">
+              <h4 class="card-title"><strong>Discussion Topic/ goal/ description</strong></h4>
+              <p class="card-text">
+                This is a space for discussion of different topics etc... Enjoy your time here and please 
+                be respectful of others in this space.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row mb-3">
+        <div class="col">
+          <ul class="nav nav-tabs">
+            <li class="nav-item">
+              <a class="nav-link active" href="/teatime.html">Comments</a>
```

Comment:
I think teatime and teatime2 aren't very descriptive names, how about tea time comment and tea time guidelines?

Reply:
ok! I'll change their names now. thank you

Reply:
I think you also need to change the filenames in links. Also, you still have your old files added in the pull request, so you want to remove them in next commit.

Keeping the main page teatime.html is fine. Just make sure you have descriptive names for comments tab and community guidelines tab (if you also need another html file for comments tab in the future)

Reply:
ok, I just added that to the new commit -- hopefully it worked.